db/migrate/20231214184439_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.references :author, foreign_key: { to_table: :users }
+      t.references :post, foreign_key: { to_table: :posts }
```

Comment:
- [ ] As illustrated in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), comment is directly related with user. **It's expected that you create `user_id` and not `author_id` as you've done properly in the migration file for likes** ℹ️